src/main/py/hl7_listener/main.py

```diff
@@ -1,22 +1,25 @@
-"""
-This HL7 MLLP Listener/Receiver Service will do the following:
-1) Connect to the configured HL7 MLLP host and then listen for incoming HL7 messages.
-2) Received messages will be sent to the configured NATS JetStream server Subject. If the message
-   send to the NATS server fails, the process of listening for incomming HL7 messages will halt.
+"""This HL7 MLLP Listener/Receiver Service will do the following: 1) Connect to the
```

Comment:
I liked it how you had it before. Maybe it has to do with no period or blank line after `following`?

Reply:
added a blank line and it looks better now